README.md

```diff
@@ -6,14 +6,53 @@
 $ bower install gaia-components/gaia-button
 ```
 
+Then include folowing files in HTML
+
+```html
+<link href="bower_components/gaia-icons/gaia-icons.css"></link>
+<script src="bower_components/gaia-list/gaia-button.js"></script>
+```
+
+You don't have to include `gaia-icon.css` if you do not use icon in your button.
+
+
 ## Examples
 
 - [Example](http://gaia-components.github.io/gaia-button/)
 
+
 ## Usage
 
+Button that contain text need to include `data-l10n-id` attribute for localization.
+
 ```html
-<gaia-button>normal button<gaia-button>
+<gaia-button><span data-l10n-id="normal_button"/><gaia-button>
+```
+
+### With icon
+
+* {Start icon} button text
+
+Button can show with icon at left
+
+```
+<button><span data-icon="back-light" data-l10n-id="back"/></button>
```

Comment:
Also not that since gaia-button will have button semantics, you can override all of its textual content with aria-label on the top element like this:
```
<gaia-button data-l10n-id="back"><span data-icon="back-light" data-l10n-id="back"/></gaia-button>
```
and have properties file as follows: 
```
back=Back
back.ariaLabel=Back
```
Screen reader will still read the aria-label only